local/src/core/YamlParser.cc

```diff
@@ -446,7 +460,7 @@ YamlParser::parse_fields_and_rules(
     }
 
     TextView name{Localizer::localize_lower(node[YAML_RULE_KEY_INDEX].Scalar())};
-    const YAML::Node ValueNode{node[YAML_RULE_VALUE_INDEX]};
+    YAML::Node ValueNode{node[YAML_RULE_VALUE_INDEX]};
```

Comment:
Why did the `const` have to drop? Scanning through, I'm not seeing the modification. I'm probably missing it...

Reply:
Originally tried overwriting this node and forgot to undo it when abandoning that approach. Fixed